compute/cker/src/train/Loss.test.cc

```diff
@@ -199,3 +278,180 @@ TEST(CKer_Operation, neg_LossMSEGrad)
                                                 nnfw::cker::Shape{2, 3}, deriv_y_pred.data()));
   }
 }
+
+TEST(CKer_Operation, LossCategoricalCrossEntropy)
+{
+  {
+    nnfw::cker::Shape in_shape{1, 10};
+    nnfw::cker::Shape out_shape{1, 1};
+
+    std::vector<float> y_pred = {2.86E-12, 2.82E-13, 0.99999845, 2.36E-07, 2.91E-16,
+                                 2.10E-07, 1.69E-14, 1.21E-17,   1.08E-06, 6.23E-18};
+    std::vector<float> y_true = {0, 0, 0, 0, 0, 0, 0, 0, 0, 1};
+    std::vector<float> expected = {39.617155};
+
+    LossCCEVerifier<float> verifier(in_shape, out_shape);
+    verifier.verifyForward(y_pred, y_true, expected);
```

Comment:
I am not against this PR. But for future, I have some questions:

1. How did you prepare `y_pred` values?
2. How did you prepare expected loss value?

Reply:
The `y_pred` value is the softmax input value of the `fashion mnist` dataset.
Then, I manually calcuated the expected loss value using  https://github.com/Samsung/ONE/issues/11248#issuecomment-1807569062 formula. Unfortunately, the log output varies slightly depending on the library, environment, etc. I used python numpy api.